release-libraries/action.yaml

```diff
@@ -21,6 +21,11 @@ inputs:
     required: true
     description: |
       Github Token needed for automatic tagging when publishing
+  fail-build:
+    required: false
```

Comment:
are you suggesting that `required: true` or that `default: true`?

my opinion is that `default: true` and `required: false` -> this will allow the action to be used without any changes to the repos that use it when they upgrade to the new charming action version, but will get the benefit of failures if their action fails to release libraries

Reply:
great spot. changed default to `true` in [8d00f81](https://github.com/canonical/charming-actions/pull/135/commits/8d00f81e3772a5264d67b4c7f3f38d38977f3aee)